packages/mdc-textfield/foundation.js

```diff
@@ -57,6 +57,7 @@ export default class MDCTextfieldFoundation extends MDCFoundation {
     this.inputBlurHandler_ = () => this.deactivateFocus_();
     this.inputInputHandler_ = () => this.autoCompleteFocus_();
     this.inputKeydownHandler_ = () => this.receivedUserInput_ = true;
+    this.useNativeValidityChecking_ = true;
```

Comment:
"Native" here refers to depending on the browser's native HTML and DOM features for form validation; I think that's reasonable, but flipping the variable name is also reasonable.